treewide: Initial role commit

haslersn opened the pull request:
TODOs:

- [x] `README.md`

haslersn committed:
treewide: Initial role commit

haslersn committed:
README.md: init

SlothOfAnarchy commented on README.md:
````diff
@@ -0,0 +1,29 @@
+  roles:
+    - role: pulseaudio
+      pulseaudio_user: stuvus
+```
````
Our [Ansible role skeleton](https://github.com/stuvusIT/ansible-skeleton) adds a section for author information, did you leave that out for a reason? It's up to you but maybe you want to have the attribution.

haslersn replied:
For project specific questions or reports we have GitHub issues.

SlothOfAnarchy replied:
I asked it here because it is the initial commit of this project? Your README.md has no author information. I mentioned it because you may have forgot to add it. Feel free to merge the role if you want.

SlothOfAnarchy reviewed (changes requested):
Please add `meta/main.yml` so the project can be indexed for http://galaxy.ansible.com/

haslersn committed:
.gitignore: Pulled from stuvusIT/ansible-skeleton

https://github.com/stuvusIT/ansible-skeleton/commit/1366fd89bd13bde1ddd836e4de59fe53b670ef3f

haslersn committed:
meta/main.yml: Needed for https://galaxy.ansible.com/
